### DISCUSSION TITLE
Temporary `ModuleNotFoundError`

### DISCUSSION THREAD

**Discussion by fweth** [codesandbox/codesandbox-client#6521] [Q&A]
Hi, I don't know why but in my recent project I get extremly often the error `Could not find module in path: 'react-firebase-hooks/auth' relative to '/src/App.js'` from the import statement, sometimes even on reload, but it always vanishes with another reload, so apparently there is no problem with the statement, what could be the cause of this?  In other projects, I never got something like this.

[EDIT] It always seems to work after the 'transpiling modules' screen appears, which is not triggered by every reload, but at least by every second reload.

**Comment by JamesACS** [comment]
Hey @fweth - These issues can sometimes be caused by a [default field added to the Json](https://stackoverflow.com/questions/60606665/how-to-remove-mysterious-default-field-added-to-json)
